compiler/test/fail_compilation/test19812b.d

```diff
@@ -0,0 +1,29 @@
+/* REQUIRED_ARGS=-preview=dip1000
```

Comment:
```suggestion
/* REQUIRED_ARGS: -preview=dip1000
```

> Test directive `REQUIRED_ARGS` must be followed by a `:`

Reply:
Dang, right in front of my face! Thanks!

(But that doesn't explain why it doesn't appear in the log.)

Reply:
It is printed to stdout, maybe it only shows stderr

Reply:
I saw it printed when I ran it locally, did you not use the test runner? I don't understand how it ran correctly on your local machine, unless you manually compiled the test.